### PR TITLE
review release

### PR DESCRIPTION
**Pull request by samcaspus**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -52,9 +51,9 @@ function transformEventToRow(fullEvent) {
     console.log(properties)
     // only move prop to elements for the $autocapture action
     if (event === '$autocapture' && properties?.['$elements']) {
```
Comment (samcaspus) [inline index.js:51]: @yakkomajuri  we are not exactly sure what needs to be done here could you give an example.

File: index.js [hunk 2]
```diff
@@ -84,9 +83,14 @@ async function exportEvents(events, { global, storage }) {
 
     rows = rows.join().split('\n')
 
-
-    const data = await storage.get('data', [])
-
+    let data = null
+    // this is an intentional change as testing storage with posthog is a little difficult to simulate
+    if (!global.tests) {
```
Comment (samcaspus) [inline index.js:85]: @yakkomajuri  Could you confirm if this is okay to have as this is the only way to add the test from what i can understand.

File: index.js (hunk 1 above)
Comment (yakkomajuri) [reply index.js:51]: this comment:

```
    // only move prop to elements for the $autocapture action
```
was written by you - not me :D

File: index.js (hunk 2 above)
Comment (yakkomajuri) [reply index.js:85]: there are utils available to mock `storage`. 

one can also mock it very easily:

```js
const storageObject = {} 

const mockedStorage = {
    get: (key, default) => storageObject[key] || default,
    set: (key, value) => { storageObject[key] = value }
}
```

**Pull request merged**